.eslintrc.js

```diff
@@ -241,3 +244,14 @@ module.exports = {
     },
   ],
 };
+
+function restrictNodeModuleImports(...modules) {
+  return modules.map((module) => ({name: module, message: `Please use 'node:${module}' instead.`}));
+}
+
+function restrictImportDestructuring(...modules) {
+  return modules.map((module) => ({
+    selector: `ImportDeclaration[source.value='${module}'] ImportSpecifier`,
+    message: `Importing from '${module}' using destructuring is restricted.`,
```

Comment:
not sure about the message here, was using `is not allowed` instead of `is restricted` but that sounded a bit harsh :)

Reply:
Seems fine, as it's just the message for our team.

Reply:
yeah but also other contributors, I am not a native english speaker but I think as it is currently should be accurate and also formally correct.